### PR TITLE
Add nsp security check

### PR DESCRIPTION
**Pull request by therealklanni**
Should not be merged until the following issues are resolved

Tracking issues:
[BitGo](/BitGo/BitGoJS) [#20](http://github.com/BitGo/BitGoJS/issues/20)
[chain-node](/chain-engineering/chain-node) [#27](http://github.com/chain-engineering/chain-node/issues/27)
[request](/request/request) [#2020](http://github.com/request/request/issues/2020)

cc @mrose17

### CODE REVIEW

**Comment by mrose17** [comment]
thanks!

File: package.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "description": "An example of client code for the Brave vault.",
   "main": "index.js",
   "scripts": {
+    "checks": "npm run check-security",
+    "check-security": "nsp check",
```
Comment (mrose17) [inline package.json]: any chance we could make this "test" and "test-security" ???

Comment (therealklanni) [inline package.json]: Sure thing

**Comment by mrose17** [comment]
fixed in https://github.com/brave/vault-client/commit/cb19e2410df55a7f4ea7eeb5c51862b1d0ac1fb3

**Pull request closed without merging**